```
=========================== short test summary info ============================
FAILED tests/test_againstmatlab.py::test_rescale_highpass - AttributeError: module 'numpy' has no attribute 'int'.
FAILED tests/test_registration.py::test_estimatereg - AttributeError: module 'numpy' has no attribute 'int'.
FAILED tests/test_sampling.py::test_rescale_lanczos - AttributeError: module 'numpy' has no attribute 'int'.
FAILED tests/test_sampling.py::test_rescale_bilinear - AttributeError: module 'numpy' has no attribute 'int'.
FAILED tests/test_sampling.py::test_rescale_nearest - AttributeError: module 'numpy' has no attribute 'int'.
FAILED tests/test_xfm3.py::test_integer_input - AttributeError: module 'numpy' has no attribute 'int'.
=========== 6 failed, 119 passed, 214 skipped, 16 warnings in 21.58s ===========
```

--- a/.gitignore
+++ b/.gitignore
@@ -17,3 +17,4 @@ cover
 # Editor backup files
 .*.swp
 *~
+.DS_Store

--- a/dtcwt/sampling.py
+++ b/dtcwt/sampling.py
@@ -35,8 +35,8 @@ DTHETA_DY_2D = np.array((_W0, _W0, _W1, -_W1, -_W0, -_W0))
 
 def _sample_clipped(im, xs, ys):
     """Truncated and symmetric sampling."""
-    sym_xs = reflect(xs, -0.5, im.shape[1]-0.5).astype(np.int)
-    sym_ys = reflect(ys, -0.5, im.shape[0]-0.5).astype(np.int)
+    sym_xs = reflect(xs, -0.5, im.shape[1]-0.5).astype(np.int32)
+    sym_ys = reflect(ys, -0.5, im.shape[0]-0.5).astype(np.int32)
     return im[sym_ys, sym_xs, ...]
 
 def _sample_nearest(im, xs, ys):
@@ -328,7 +328,7 @@ def _upsample_columns(X, method=None):
     
     # Convolve
     for di, l_a, l_b in zip(sample_offsets, l_as, l_bs):
-        columns = reflect(int_columns + di, -0.5, M-0.5).astype(np.int)
+        columns = reflect(int_columns + di, -0.5, M-0.5).astype(np.int32)
         
         output[:,0::2,...] += l_a * X[:,columns,...]
         output[:,1::2,...] += l_b * X[:,columns,...]

--- a/tests/test_xfm3.py
+++ b/tests/test_xfm3.py
@@ -123,7 +123,7 @@ def test_simple_level_4_recon_ext_mode_4():
 def test_integer_input():
     # Check that an integer input is correctly coerced into a floating point
     # array
-    Yl, Yh = dtwavexfm3(np.ones((4,4,4), dtype=np.int))
+    Yl, Yh = dtwavexfm3(np.ones((4,4,4), dtype=np.int32))
     assert np.any(Yl != 0)
 
 def test_integer_perfect_recon():

